navigation/screens/review-form.tsx

```diff
@@ -11,14 +11,15 @@ import { createReview } from "@/controllers/reviewController";
 import Main from "./home";
 import { useNavigation } from "@react-navigation/native";
 
-export default function TabReviewForm() {
+export default function TabReviewForm(
+  username: string,
+  bathroomID: string,
+  time: Date,
+  images: string[]
```

Comment:
please remove this line

images should not be passed into the constructor because we won't have any before this component is created. this component contains the image picker